index.html

```diff
@@ -4,9 +4,77 @@
+ 
+    <!-- Bootstrap CSS-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"/>
+    
     <title>Hymetamedia</title>
 </head>
 <body>
-    
+    <!--Start header -->
+    <div class="off-screen">
+        <div class="container">
+            <ul class="menu">
+                <li class="menu-item"><a href="#">Home</a></li>
+                <li class="menu-item"><a href="#">Service</a></li>
+                <li class="menu-item"><a href="#">Contact</a></li>
+                <li class="menu-item"><a href="#">Careers</a></li>
+            </ul>
+        </div>
+        <div class="bottom">
+            <p class="bottom-text">
+                <a href="#" class="a-tag">VX | </a> 
+                <a href="#" class="a-tag">Privacy Policy | </a>
+                <a href="#" class="a-tag">Terms | </a> 
+                <a href="#" class="a-tag">Copyright Policy</a>
+            </p>
+            <div class="bottom-icon">
+                    <a href="#" class="a-tag"><i class="fa-brands fa-twitter"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-facebook-f"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-pinterest"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-linkedin-in"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-youtube"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-instagram"></i></a>
+                    <a href="#" class="a-tag"><i class="fa-brands fa-square-snapchat"></i></a>
+            </div>
+        </div>
+    </div>
+    <nav class="navbar fixed-top navbar-expand-lg py-3">
+        <div class="container justify-content-between align-items-center">
+            <a href="">
+                <img src="./assets/img/vaynermedia-black-on-transparent.png" class="company-logo" alt="Logo">
+            </a>
+            <div id="hamburger-menu">
+                <div class="ham-bar bar-top"></div>  
+                <div class="ham-bar bar-mid"></div>  
+                <div class="ham-bar bar-bottom"></div>  
+            </div>
+        </div>
+    </nav>
+    <!-- End Header -->
+
+    <!-- Start banner -->
+    <div class="container-wrap banner">
+        <div class="banner-image">
+            <div class="container">
+                <img src="./assets/img/adweeklogowhite.png" alt="adWeekLogoWhite" class="adweekImg">
+                <p class="agency-text">
+                    <mark>MEDIA AGENCY OF THE YEAR</mark>
+                </p>
+                <p class="story-text">" THIS IS A STORY OF <br> PIRATES..."</p>
```

Comment:
Always use text in smaller letters and use css property `text-transform: uppercase` to make them all uppercase.